test/invoice_parser_test.rb

```diff
@@ -4,6 +4,6 @@
 
 class InvoiceParserTest < MiniTest::Test
   def test_it_exists
-    assert InvoiceParser.new('test/support/sample_invoices.csv')
+    assert InvoiceParser.new('test/support/invoices.csv')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.